src/components/calendar/FullscreenCalendar.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import { View, Text, Dimensions } from 'react-native'
+import { DateData } from 'react-native-calendars'
+import { format, eachDayOfInterval } from 'date-fns'
+import ScrollingCalendar from '@/components/calendar/ScrollingCalendar'
+import Button from '@/components/pressables/Button'
+import { StartEndDates } from '@/types/dates'
+import { ButtonType, ButtonVariant } from '@/types/button'
+import Icon from '../Icon/Icon'
```

Comment:
Did we wanna use @/components/Icon/Icon ?

Reply:
👍